web/src/components/AppBar/ApiKeyItem.vue

```diff
@@ -3,7 +3,7 @@
     <v-list-item-action class="mr-2">
       <v-btn
         icon
-        @click="reloadApiKey"
+        @click.stop="fetch"
```

Comment:
I realized that the `.stop` here may be worth adding a code comment, explaining why we do this. If you only look at this one component, it's not at all obvious.

Relatedly, see: https://stackoverflow.com/a/41229144